package.json

```diff
@@ -35,13 +35,16 @@
     "@types/semver": "^6.0.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
+    "ncp": "^2.0.0",
     "husky": "^1.3.1",
     "lint-staged": "^8.1.5",
+    "mkdirp": "^0.5.1",
     "mocha": "^6.1.3",
     "prettier": "^1.16.4",
     "require-npm4-to-publish": "^1.0.0",
     "resin-lint": "^2.0.1",
-    "tar-utils": "^1.1.0",
+    "rimraf": "^2.6.3",
+    "tar-utils": "^2.0.0",
```

Comment:
All these changes to `package.json` are to allow `npm test` to pass on Windows (where the `rm` and `cp` commands are not available by default). Now the tests pass! I wanted to run the tests on Windows check issues with path separators, as this may be related to these other issues:
* https://forums.balena.io/t/deploy-failed-with-error-copy-failed-no-such-file-or-directory/6041
* https://github.com/balena-io/balena-cli/issues/1133